### DISCUSSION TITLE
slack slash command returns 'failed with the error "dispatch_failed"'

### DISCUSSION THREAD

**Discussion by FurkanTheHuman** [outline/outline#3163] [Self Hosting]
I have installed Outline to our kubernetes cluster. Also, I configured the slash command '/outline' for slack to use. [With this doc as the reference](https://app.getoutline.com/share/770a97da-13e5-401e-9f8a-37949c19f97e/doc/slack-G2mc8DOJHk). Login with slack works but I can not run the slash command. It gives 
```
/outline failed with the error "dispatch_failed"
```

And this seems like a generic error. I tried to run slack in debug mode to gather more info with 
```
$ slack -g debug -x

[SLASH-COMMAND] (TF0GEHQHF) Looks like a slash command; attempt to parse and execute
[SLASH-COMMAND] (TF0GEHQHF)) Command "/outline" is PENDING
[API-Q] (TF0GEHQHF)) 5d495e2d-XXXXX.109 chat.command called with reason: executeCommand
[API-Q] (TF0GEHQHF)) 5d495e2d-XXXXX.109 chat.command is ENQUEUED
[UNREAD-COUNTS] (TF0GEHQHF)) Counting unreads in D02BT4LFLPP because message removed
[API-Q] (TF0GEHQHF)) 5d495e2d-XXXXX.109 chat.command is ACTIVE
[API-Q] (TF0GEHQHF)) 5d495e2d-XXXXX.109 chat.command is REJECTED: API not ok dispatch_failed
[SLASH-COMMAND] (TF0GEHQHF)) Command "/outline" FAILED (server error "dispatch_failed")

```
I guess this doesn't provide any valuable information but putting it anyway.  

Any ideas on why?

**Comment by warnus** [comment]
Anybody help this question? I met same problem. https://github.com/outline/outline/discussions/3086

**Comment by nbently** [comment]
@FurkanTheHuman if you're still having issues with this, try setting the `SLACK_VERIFICATION_TOKEN` env variable using the verification token in Basic Information in your Slack App settings. Seemed to do the trick for me.

**Reply by antonprakash** [reply]
Without all that, a simple webhook created with webhook.site worked. But an AWS API Gateway/Lambda endpoint (with or without the /slack/events route) results in this error.
